OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs

```diff
@@ -72,6 +73,13 @@ public bool HasProperty(string name)
 		{
 			return Self.HasScriptProperty(name);
 		}
+
+		[Desc("Render a target flash on the actor. If set, 'asPlayer'",
```

Comment:
Are these strings joined together with a space or directly concatenated?
If the latter you'll want to insert a space at the end of the first string or at the beginning of the second.